lib/less/rails/railtie.rb

```diff
@@ -16,7 +16,7 @@ class Railtie < ::Rails::Railtie
       end
 
       initializer 'less-rails.before.load_config_initializers', :before => :load_config_initializers, :group => :all do |app|
-        app.assets.register_preprocessor 'text/css', ImportProcessor
+        app.assets.register_preprocessor 'text/css', ImportProcessor if app.assets
```

Comment:
I think we should do it in this way to support both versions in one step and apply this to all `app.assets` calls.:

```ruby
sprockets_env = app.assets || app.config.assets
sprockets_env.register_preprocessor 'text/css', ImportProcessor
```